.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Types of changes
```

Comment:
does this file appear somewhere with the bot? I don't think we need this. I thought we had contributing guidelines online that we could reference.

Reply:
I can get rid of this template for sure, so far as I know we don't have specifics on what should go into a PR anywhere on the site. The [building the website](http://pyne.io/devsguide/website.html#procedure-for-modifying-the-website) page has a procedure for modifying the website, but it doesn't have details for PRs